src/chess.ts

```diff
@@ -2202,9 +2238,13 @@ export class Chess {
     }
 
     // generate the FEN for the 'after' key
-    this._makeMove(uglyMove)
-    move.after = this.fen()
-    this._undoMove()
+    if (!(uglyMove.flags & BITS.NULL_MOVE)) {
+      this._makeMove(uglyMove)
+      move.after = this.fen()
+      this._undoMove()
+    } else {
+      move.after = this.fen()
+    }
```

Comment:
Not sure this change is needed, don't we need to make the null move and return the new fen with different color and move number etc?